src/compiler/factory/emitHelpers.ts

```diff
@@ -395,6 +397,10 @@ namespace ts {
             return factory.createCallExpression(getUnscopedHelperName("__classPrivateFieldSet"), /*typeArguments*/ undefined, args);
         }
 
+        function createClassPrivateFieldInHelper(receiver: Expression, state: Identifier) {

```

Comment:
I recommend swapping the order of `receiver` and `state`, per my comments below on `classPrivateFieldInHelper`.